### DISCUSSION TITLE
Feature: Scenario Editor modifications

### DISCUSSION THREAD

**Discussion by Dracheron** [OpenRCT2/OpenRCT2#15781] [Feature requests]
Something that has dawned upon me for a bit is that ever since some modifications have been done with research in scenarios it has been easier to get coasters in scenarios where money is not an object.  As a result some of those scenarios have become significantly easier since you can get the stuff you want faster and skip things you don't want.  What I propose is an option forbidding the player to change research settings under any conditions - I would probably either make it part of the park options in the scenario editor or even be its own tab in the scenario editor.  This would give such scenarios their due difficulty from the original without making a massive modification to the game's code by making it forbidden to skip the order presented in the scenario editor.

**Comment by IntelOrca** [comment]
Well, I suppose (after we have a new save format) we can add a new park flag / scenario setting to forbid changes to research priorities.

**Comment by Gymnasiast** [comment]
> **Something that has dawned upon me for a bit is that ever since some modifications have been done with research in scenarios it has been easier to get coasters in scenarios where money is not an object.**

Which modifications are you talking about?

**Comment by IntelOrca** [comment]
I think they mean that you can now access the research window when money is disabled. You could not in vanilla and were therefore locked to a specific research order.

**Comment by Dracheron** [comment]
@Ted John is right on this one - and it wasn't just the order - the speed
is also modified in openrct due to this change as well iirc

**Comment by Gymnasiast** [comment]
> the speed is also modified in openrct due to this change as well iirc

No, it isn't.

**Comment by IntelOrca** [comment]
It is possible that a scenario may have had funding set to minimal, in OpenRCT2 we always force it to normal if money is disabled.

**Comment by Gymnasiast** [comment]
@IntelOrca I haven't found any code that does that. Only code that handles the "no funding at all" case for no-money scenarios.

**Comment by IntelOrca** [comment]
Yeah sorry, we only did that if the scenario had no funding set.
